index.test.js

```diff
@@ -217,5 +275,16 @@ describe("raven-for-redux", () => {
       // Even though the action isn't added to breadcrumbs, it should be sent with extra data
       expect(extra.lastAction).toEqual({ type: "UNINTERESTING_ACTION" });
     });
+    it("preserves user context", () => {
+      const userData = { userId: 1, username: "captbaritone" };
+      Raven.setUserContext(userData);
+      expect(() => {
+        context.store.dispatch({ type: "THROW", extra: "BAR" });
+      }).toThrow();
+
+      expect(context.mockTransport.mock.calls[0][0].data.user).toEqual(
+        userData
+      );
+    });
```

Comment:
I don’t think this test is really nessesary. The first one like this, in the “default config” test block is probably enough.

Reply:
Removed the "duplicate" test now